python/engine/test/BadMeasure/measure.py

```diff
@@ -0,0 +1,33 @@
+import typing
+
+import openstudio
+
+
+class BadMeasure(openstudio.measure.ModelMeasure):
+    def name(self):
+        return "Bad Measure"
+
+    def modeler_description(self):
+        return "The arguments method calls another_method which does a raise ValueError"
+
+    def another_method(self):
+        raise ValueError("oops")
+
+    def arguments(self, model: typing.Optional[openstudio.model.Model] = None):
+        self.another_method()
```

Comment:
Both Python and Ruby have this failed measure, that calls another method that throws, so we can check if the traceback is good